app/components/businesses/upgrade_required_component.rb

```diff
@@ -11,7 +11,7 @@ def render?
     end
 
     def expired?
-      !permission.active_subscription?
+      !permissions.active_subscription?
```

Comment:
```suggestion
      !user.permissions.active_subscription?
```

Then we can remove the private helper method, too.